index.bs

```diff
@@ -4932,6 +4932,27 @@ partial interface MLGraphBuilder {
     </div>
 </details>
 
+<div class="note">
+  <details open>
+    <summary>
+    The behavior of several reduction operations can be generically emulated from the usage of
+    other operations as follows. However, user agents typically have a more
+    efficient implementation for them, therefore their usage is encouraged from the
+    performance standpoint.
+    </summary>
+    <pre highlight="js">
+    // reduceLogSum(input, options)
+    return builder.log(builder.reduceSum(input, options));
+
+    // reduceLogSumExp(input, options)
+    return builder.log(builder.reduceSum(builder.exp(input), options));
```

Comment:
Yep, looks correct to me.

And I always have to go double-check this one because the name is...

 `reduce`|`log`|`sum`|`exp`

`reduce`
`log`
`sum`
`exp`

...but actual operator order is:

`a = exp(input)`
`b = reduceSum(a)`
`c = log(b)`